src/Tickets/Commerce/Gateways/PayPal/Gateway.php

```diff
@@ -128,4 +128,21 @@ public function get_logo_url() {
 	public function get_subtitle() {
 		return __( 'Enable payments through PayPal, Venmo, and credit card', 'event-tickets' );
 	}
+	
+	/**
+	 * @inheritDoc
+	 */
+	public static function is_enabled() {
+		if ( ! static::should_show() ) {
+			return false;
+		}
+		
+		$option_value = tribe_get_option( static::get_enabled_option_key() );
```

Comment:
Should we have the default as false?

Reply:
I mean, what would happen with a site that is installing ET for the first time?

Reply:
The default is '' when you don't pass one, so I check for '' explicitly in the next line. I'm actually looking for the scenario of when the `enabled` flag isn't explicitly set. That would include first installs and upgrades to the newest version, since that flag didn't exist. I guess checking `false === $option` would serve the same purpose, but I just chose the one that had less code.

Reply:
Perfect!